### DISCUSSION TITLE
Azure has released the v2 API

### DISCUSSION THREAD

**Discussion by kenleren** [RageAgainstThePixel/OpenAI-DotNet#313] [Q&A] [answered]
Great news for us Europeans who need to consider GDPR! 

Will V2 support for Azure OAI services be included in 8.0.0?

https://learn.microsoft.com/en-us/azure/ai-services/openai/whats-new
https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/announcing-azure-openai-service-assistants-public-preview/ba-p/4143217

".NET support will be available in June" - In reference to their SDK.

Ever grateful for your work!

**Comment by ADefWebserver** [comment]
> Ever grateful for your work!

👍🏽🙏🏽

**Reply by StephenHodgson** [reply, marked as answer]
@kenleren most support should have been added in https://github.com/RageAgainstThePixel/OpenAI-DotNet/releases/tag/v8.2.0
